x-pack/legacy/plugins/siem/public/store/network/reducer.ts

```diff
@@ -98,13 +104,26 @@ export const initialNetworkState: NetworkState = {
 };
 
 export const networkReducer = reducerWithInitialState(initialNetworkState)
+  .case(updateTableActivePage, (state, { activePage, networkType, tableType }) => ({
+    ...state,
+    [networkType]: {
+      ...state[networkType],
+      queries: {
+        ...state[networkType].queries,
+        [tableType]: {
+          ...state[networkType].queries[tableType],
```

Comment:
I am having a very hard time with this type. Below is the error:
```
x-pack/legacy/plugins/siem/public/store/network/reducer.ts:114:14 - error TS7053: Element implicitly has an 'any' type because expression of type 'NetworkTableType' can't be used to index type 'NetworkQueries | IpOverviewQueries'.
Property '[NetworkTableType.dns]' does not exist on type 'NetworkQueries | IpOverviewQueries'.
114           ...state[networkType].queries[tableType],
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Found 1 error.
```

Reply:
@XavierM @spong I see you guys worked on these types, if you have time I'd appreciate your help :)

Reply:
thansk @spong for some zoom help :)